src/ui/helper/index.ts

```diff
@@ -365,3 +365,15 @@ export function findElByClassName(e: Event, className: string) {
     target : findParentBySelector(target, '.' + className)
 }
 
+export function externalLink(text: string, url: string): Mithril.Children {
+  return h('span', { class: 'link' }, [
```

Comment:
No need to have an `a` inside a `span`. Also no need for class `link`, so remove the span element.